services/transcribe.py

```
@@ -126,7 +127,39 @@ def download_and_transcribe(self, video_url: str, clip_id: str) -> tuple[str, Pa
     
     def _download_video(self, url: str, output_path: Path):
-        """Download video from URL."""
+        """Download video from URL using yt-dlp for Twitch clips."""
+        try:
+            # Use yt-dlp for Twitch clips, regular download for other URLs
+            if 'clips.twitch.tv' in url or 'twitch.tv' in url:
+                return self._download_with_ytdlp(url, output_path)
+            else:
+                return self._download_with_httpx(url, output_path)
+                            
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)